app/controllers/admin/themes_controller.rb

```diff
@@ -287,6 +300,7 @@ def destroy
   def bulk_destroy
     themes = Theme.where(id: params[:theme_ids])
     raise Discourse::InvalidParameters.new(:id) if themes.blank?
+    raise Discourse::InvalidAccess.new(:id) if themes.any? { |theme| theme.system? }
```

Comment:
Similar to `https://github.com/discourse/discourse/pull/32681/files#r2139346510`, We can reject negative values from `params[:theme_ids]`.